config/routes.rb

```diff
@@ -1377,6 +1377,24 @@
       )
     },
 
+    :physical_chassis    =>  {
+      :get  =>  %w(
+        download_data
+        perf_top_chart
+        protect
+        show_list
+        show
+      ),
+
+      :post   =>  %w(
+        button
+        show_list
+        quick_search
+        create
```

Comment:
you have `create` and `update` actions in your controller? I don't see those.

Reply:
You're right. We don't have and probably will never have those actions.